transport/dlr-malspp/transport/src/main/java/org/ccsds/moims/mo/testbed/util/sppimpl/util/SPPReader.java

```diff
@@ -138,6 +179,7 @@ public synchronized SpacePacket receive() throws IOException
     outPacket.setLength(dataLength);
     byte[] data = new byte[outPacket.getLength()];
     read(data, outPacket.getOffset(), dataLength);
+
     outPacket.setBody(data);
```

Comment:
Packet length should not depend on crcEnabled, but crcEnabled && crcApids.inRange(apid)

Reply:
But even if I don't process the CRC, I still need to get rid of the last two bytes so the CRC is not put into the body, don't I?